### ISSUE TITLE
fi fi-rr-english and many other language related icons are missing

### ISSUE THREAD

**Issue by tiberiu-potec** [freepik-company/flaticon-uicons#30]
Hi, 

I'm really enjoying your package, however I've found some of the icons in languages section won't load.
`fi fi-rr-english` is one of them, but there are many others. I guess it's the ones from version `2.5.0` on the website.
Can you please update the library with latest icons?

Thanks

**Comment by NEDDL** [comment]
@tiberiu-potec I agree that there are so many missing icons that needs to be added. I cant reach many of them. Any development on this topic? @konchimartin 

Some that didn't work for me:
fi-sr-legal-case
fi-sc-cookie
fi-sr-confidential-discussion
